.gitignore

```diff
@@ -8,6 +8,7 @@
 /dev-tools/infection/
 /dev-tools/php-compatibility/cache.json
 /dev-tools/phpstan/cache/
+/dev-tools/phpunit/cache/
```

Comment:
should we now remove `/.phpunit.result.cache` from .gitignore, or both entries needed?

Reply:
good catch